src/chain_spec.rs

```diff
@@ -504,7 +517,15 @@ fn alcyone_testnet_genesis(
             code: alcyone::WASM_BINARY.to_vec(),
             changes_trie_config: Default::default(),
         }),
-        asset: Some(AlcyoneConfig::AssetConfig {
+        asset: {
+
+            let mut file = File::open("~/src/data/currency_symbols.json").unwrap();
```

Comment:
Please list the currency symbols in the chain spec, not in an external file.

Reply:
I have added the currency symbols to chain spec locally would do a push soon